pom.xml

```diff
@@ -92,6 +92,10 @@
 
     <version.org.drools.guvnor-api>5.6.0.Final</version.org.drools.guvnor-api>
     <version.org.jboss.errai>4.0.1-SNAPSHOT</version.org.jboss.errai>
+
+    <!-- Required for kie-wb-common to build in IntelliJ Idea 2017 -->
+    <version.org.jboss.errai.wildfly>10.0.0.Final</version.org.jboss.errai.wildfly>
```

Comment:
I imagine we'll want to remove this as soon as the IntelliJ bug is fixed, so I think the comment is a good idea.